### ISSUE TITLE
Error on handlePaste on chrome

### ISSUE THREAD

**Issue by jordyvanderpoel** [NoelOConnell/quill-image-uploader#88]
Using `quill-image-uploader@1.2.2`, when pasting an image from clipboard using Google Chrome, I get an error:
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'index')
    at e.value (quill.imageUploader.min.js:1:5838)
    at quill.imageUploader.min.js:1:5464
```
which seems to be `this.range` being `null` at `insertToEditor`.
This issue appears with different users, but only seems to appear in the browser Google Chrome.

**Comment by tmKnight01** [comment]
Hi,
  I also meet this problem,  how do you  resolve it ?

**Comment by jordyvanderpoel** [comment]
The issue seemed to have been resolved in release 1.2.4, updating to that version worked for me.
